ambry-clustermap/src/main/java/com.github.ambry.clustermap/ClusterMapUtils.java

```diff
@@ -122,6 +132,21 @@ public static String getInstanceName(String host, Integer port) {
     return dataCenterToZkAddress;
   }
 
+  /**
+   * Create a instance of {@link HelixPropertyStore} based on given {@link HelixPropertyStoreConfig}.
+   * @param zkServers the ZooKeeper server address.
+   * @param propertyStoreConfig the config for {@link HelixPropertyStore}.
+   * @param subscribedPaths a list of paths to which the PropertyStore subscribes.
+   * @return the instance of {@link HelixPropertyStore}.
+   */
+  public static HelixPropertyStore<ZNRecord> createHelixPropertyStore(String zkServers,
+      HelixPropertyStoreConfig propertyStoreConfig, List<String> subscribedPaths) {
```

Comment:
This is because the `zkServer` (a.k.a `zkClientConnectString` in config) can be different from that in `HelixPropertyStoreConfig`, especially in some unit tests. That's why I didn't pick zk path from `HelixPropertyStoreConfig`. Since the `zkClientConnectString` is a general config which is better to be in `HelixPropertyStoreConfig`, I prefer to pass the args separately.

Reply:
No, I are not. The property store for account service could be explicitly specified by `AccountServiceConfig.zkClientConnectStr`. They are still decoupled.